src/python/pants/pantsd/pants_daemon.py

```diff
@@ -82,6 +82,17 @@ def buffer(self):
     return self
 
 
+class PantsDaemonSignalHandler(SignalHandler):
+
+  def __init__(self, services):
+    super(PantsDaemonSignalHandler, self).__init__()
+    self._services = services
+
+  def handle_sigint(self, signum, _frame):
+    for service in self._services:
+      service.record_exception(KeyboardInterrupt('remote client sent control-c!'))
```

Comment:
This was a test for an api that I forgot to remove, it doesn't actually do anything, this line can be safely removed.